pkg/server/commands/check_command.go

```diff
@@ -34,6 +52,7 @@ type CheckQuery struct {
 
 	resolveNodeLimit   uint32
 	maxConcurrentReads uint32
+	parentMethodName   string
```

Comment:
would `caller` be a more illustrative name for this?

Reply:
callerMethodName?

Reply:
I like either of your suggestions more than `parent` :)